src/app/pages/pages.component.ts

```diff
@@ -12,6 +14,16 @@ import { MENU_ITEMS } from './pages-menu';
   `,
 })
 export class PagesComponent {
+  menu: NbMenuItem[];
 
-  menu = MENU_ITEMS;
+  constructor(private translateService: TranslateService) {
+    this.getTranslation();
+    this.translateService.onLangChange.subscribe((event: LangChangeEvent) => {
+      this.getTranslation();
+    });
+  }
+
+  getTranslation() {
+    this.translateService.currentLang === 'ar-EG' ? this.menu = MENU_ITEMS_AR : this.menu = MENU_ITEMS_EN;
```

Comment:
Would there be ways to get this to scale better with other languages? Hardcoding `ar-EG` here and in other places seems risky

Reply:
The requirements was only for Arabic and English. Of course it could be handled by adding switch case

Reply:
I think this template is being used by people all over the world, and the goal is to make it easy for people to start using it and build off their own custom components while reusing some of the heavy-lifting provided by nebular. Again, in my opinion, it should be as easy as possible for people to start using it, and I think that hardcoded strings which have potential side effects make this harder rather than easier

Reply:
@dizco I will try to implement [this gist](https://gist.github.com/dizco/0182af54fa90e9b1e4d7751c60a2a787) 

and I hope you will review it as soon I finish it. 😅😎